components/director/internal/repo/create.go

```diff
@@ -47,9 +47,11 @@ func (c *universalCreator) Create(ctx context.Context, dbEntity interface{}) err
 		values = append(values, fmt.Sprintf(":%s", c))
 	}
 
+	log.Debug("Building DB query...")
```

Comment:
Actually the statements between Building DB Query and Executing DB Query are just one fmt.Sprintf which cannot fail. Of course it can build wrong query and this is seen in the Executing... log. It looks like this log is unnecessary.

Valid for the whole change.

Reply:
Yes, in this case, we can remove the log but also we have cases where after the debug log we have a function and maybe it's a good idea to have such a debug log(at least for traceability)?